config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -28,10 +31,9 @@ presubmits:
       testgrid-num-failures-to-alert: '10'
       testgrid-alert-stale-results-hours: '24'
       testgrid-create-test-group: 'true'
-  # TODO(krzyzacy): add branches once this works
-  # TODO(krzyzacy): Consider combine kubekins-e2e and kubekins-test
-  - name: pull-kubernetes-integration-podutil
-    always_run: false
+  # TODO: replace the bootstrap / scenario based job going forward
+  - name: pull-kubernetes-integration
```

Comment:
I assume we'll need an identical change for ci-kubernetes-integration-master once the go bump PR merges

Reply:
yes

Reply:
Should this job have the branching annotations? Should it also be changed for the 1.17 branch that is already cut?

Reply:
once the bump PR merges, we'll need to update the master and 1.17 jobs

Reply:
1.17 should be changed, I don't know the branching annotations yet.. but we definitely need a follow up PR to change the CI jobs as well.

Reply:
sgtm